platforms/documentation/docs/src/docs/userguide/redirects/organizing_build_logic.adoc

```diff
@@ -0,0 +1,3 @@
+++++
+<meta http-equiv="refresh" content="0;URL=organizing_gradle_projects.html">
```

Comment:
🤔 `organizing_gradle_projects` also doesn't show up in the navigation sidebar

Reply:
This is on purpose.